packages/scripts/config/webpack.config.js

```diff
@@ -235,6 +235,24 @@ const config = {
 					from: copyWebpackPatterns,
 					context: process.env.WP_SRC_DIRECTORY,
 					noErrorOnMissing: true,
+					transform( content, absoluteFrom ) {
+						if ( basename( absoluteFrom ) === 'block.json' ) {
+							const blockJson = JSON.parse( content.toString() );
+							[ 'viewScript', 'script', 'editorScript' ].forEach(
```

Comment:
We added support for multiple entries in `viewScript`, example: `[ 'file:view.js', 'my-view-script' ]`. We plan to follow up with the same changes for `script` and `editorScript` so we have to account for that. The value can be a string or an array of strings.

Reply:
@gziolo 

Thank you for your reviewing!
I Changed to support stinrg and array of strings.

Reply:
Great! It should do the job 👍